src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ByteArrayModelBinder.cs

```diff
@@ -19,6 +41,8 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
                 throw new ArgumentNullException(nameof(bindingContext));
             }
 
+            logger.TryingToBindModel(bindingContext);
```

Comment:
Would we follow the if-elses in this type (like we do in BodyModelBinder)? This initial log has no follow up that tells if this binder succeeded.

Reply:
Take a look at my response to @dougbu comment 'Where's Done logged?'